code/game/verbs/ooc.dm

```diff
@@ -46,15 +46,33 @@ var/global/normal_ooc_colour = "#002eb8"
 	var/display_colour = normal_ooc_colour
 	if(holder && !holder.fakekey)
 		display_colour = "#2e78d9"	//light blue
-		if(holder.rights & R_MOD && !(holder.rights & R_ADMIN))
-			display_colour = "#184880"	//dark blue
-		if(holder.rights & R_DEBUG && !(holder.rights & R_ADMIN))
-			display_colour = "#1b521f"	//dark green
-		else if(holder.rights & R_COLOR)
-			if(config.allow_admin_ooccolor)
-				display_colour = src.prefs.ooccolor
-			else
-				display_colour = "#b82e00"	//orange
+		if(holder.rank == "Mentor")
+			display_colour = "#008000"	//green
+		else if(holder.rank == "Headmentor")
```

Comment:
Use `switch` instead of long `else if` chains whenever you are comparing a single var, such as, in this case, `holder.rank`